src/main/resources/hudson/plugins/copyartifact/Messages.properties

```diff
@@ -1,3 +1,4 @@
+CopyArtifact.AncestorIsNull=Ancestor/Context Unknown: the project specified cannot be validated
```

Comment:
I don't think this is helpful for users.
How about a text like "You look configuring non-buildable job (may be template) ..."?